tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -352,4 +352,67 @@ bool EnergyPlusFixture::process_idf(std::string const &idf_snippet, bool use_ass
     return successful_processing;
 }
 
+bool EnergyPlusFixture::process_idf(std::string_view const idf_snippet, bool use_assertions)
```

Comment:
That's not what I meant. I meant for this to take a `gsl::span<std::string_view>` so that you wouldn't have to concatenate all of those constant strings on the heap only to break them up again.

Reply:
I recall that now.  But I'm curious, does this take it one step further and avoid ever having all the individual strings anyway?  Just one stream of inputs for the input processor?  Ultimately this is passing the string_view all the way down to `IdfParser::parse_idf` which is just tokenizing the text from one long string regardless of how it got there.

Reply:
I guess you're right.

Reply:
OK cool.  I am planning on just letting CI verify things are good here and then merge this in.  I will put in a little effort to try to get all the unit tests updated to the single string_view technique.